pkg/tfhelmfile/resource_release.go

```diff
@@ -129,6 +130,10 @@ func resourceShellHelmfileRelease() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			KeyDiffError: {
```

Comment:
Could you please explain why it's set in release, but not in releaseSet. Maybe I'm missing something.

Reply:
Ah good catch. I had updated wrong place... Fixed in the last commit.